shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala

```diff
@@ -1056,6 +1079,7 @@ class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
       sharedConf: Broadcast[Map[String, String]]) {
 
     val conf: SQLConf = getConfFromMap(sharedConf)
+    val bytesAllowedPerBatch = new RapidsConf(conf).bytesAllowedPerBatch
```

Comment:
No need to build an entire `RapidsConf` just to get one conf setting from it.
```suggestion
    val bytesAllowedPerBatch = RapidsConf.TEST_CACHE_BATCH_SIZE.get(conf)
```